xbmc/GUIInfoManager.cpp

```diff
@@ -627,6 +627,22 @@ const infomap integer_bools[] =  {{ "isequal",          INTEGER_IS_EQUAL },
 ///     <p><hr>
 ///     @skinning_v18 **[New Infolabel]** \link Player_Icon `Player.Icon`\endlink
 ///     <p>  
+///   \table_row3{   <b>`Player.Cutlist`</b>,
```

Comment:
This breaks our doxygen guiinfo table. Please make sure you close the brackets for any new item and that commas are escaped (since commas are a new cell in the table. You removed the `}` belonging to `Player.Icon`. The correct block to save you time (added in the end of the existing one):

```
///   \table_row3{   <b>`Player.Cutlist`</b>,
///                  \anchor Player_Cutlist
///                  _string_,
///     @return The cutlist of the currently playing item as csv in the format start1\,end1\,start2\,end2\,...
///              Tokens must have values in the range from 0.0 to 100.0. end token must be less or equal start token.
///     <p><hr>
///     @skinning_v19 **[New Infolabel]** \link Player_Cutlist `Player.Cutlist`\endlink
///     <p>
///   }
///   \table_row3{   <b>`Player.Chapters`</b>,
///                  \anchor Player_Chapters
///                  _string_,
///     @return The chapters of the currently playing item as csv in the format start1\,end1\,start2\,end2\,...
///              Tokens must have values in the range from 0.0 to 100.0. end token must be less or equal start token.
///     <p><hr>
///     @skinning_v19 **[New Infolabel]** \link Player_Chapters `Player.Chapters`\endlink
///     <p>
///   }
```

Reply:
Ooops, sorry. Will fix that.

Reply:
done